README.md

```diff
@@ -68,17 +52,18 @@ go install github.com/u-root/u-root
 **Note: The `u-root` command will end up in `$GOPATH/bin/u-root`, so you may
 need to add `$GOPATH/bin` to your `$PATH`.**
 
-You can now use the u-root command to build an initramfs. Here are some
-examples with $UROOT_PATH being the path to where the u-root sources are on the disk
-(explicitly specifiying this is only necessary if not runnig u-root inside the root of the repository):
+You can now use the u-root command to build an initramfs. Here are some examples
```

Comment:
I am not sure why 'now' is in there, since the initramfs has been in there for-f-ing ever :-) Maybe I did that.